Aggregate bounded split in transaction list

decentraliser opened the pull request.

aleixmorgadas commented:
Could you share a little description about the `aggregate-bonded` splitting? I cannot fully understand what the code does.

decentraliser commented:
sure!
I just noticed that the word "bounded" should not be there, I split all the aggregate tx there.

so first, I believe it is opinionated from me. If you want to display aggregate tx in a list, you have 2 choices: or you display one line, that shows 

1. aggregate transaction: transaction number
2. you show a spread of the inner transactions

I went for option 2, as I think it is much more enjoyable UI-wise

Basically, I added an extra pipe to the transaction formatting: if a transaction has a `innerTransactions` field, then it is an aggregate tx. So instead of formatting the aggregate tx itself, we format the inner transactions.

`export const formatTransactions = (tx) => {
    if (tx.innerTransactions) {
        return tx.innerTransactions.map((t) => {
            // eslint-disable-next-line no-param-reassign
            t.aggregate = txTypeNameFromTypeId(tx.type);
            return formatTransaction(t);
        });
    }
    return [formatTransaction(tx)];
};`
